### ISSUE TITLE
No pool statistics in UI after provisioning a volume with StorageClass "name": rpc error: code = Internal desc = Not enough space in pool PoolX 

### ISSUE THREAD

**Issue by TymoT** [IBM/ibm-storage-odf-console#41]
# After adding a additional  FlashSystem Storage Class and creating PVC with it the PV creation failing due to "Not enough space in pool PoolX" error pool statistics in ODF UI are not showed properly:

Created PVC with too big PV size:
```
Events:
  Type     Reason                Age                  From                                                                               Message
  ----     ------                ----                 ----                                                                               -------
  Warning  ProvisioningFailed    13m (x15 over 41m)   block.csi.ibm.com_ibm-block-csi-controller-0_c89c8a35-02f6-4703-8ad3-558376e55e36  failed to provision volume with StorageClass "flashsystem-sc-pool6": rpc error: code = Internal desc = Not enough space in pool Pool6
  Normal   Provisioning          3m9s (x18 over 41m)  block.csi.ibm.com_ibm-block-csi-controller-0_c89c8a35-02f6-4703-8ad3-558376e55e36  External provisioner is provisioning volume for claim "openshift-storage/pvc-test05-pool6"
  Normal   ExternalProvisioning  80s (x167 over 41m)  persistentvolume-controller                                                        waiting for a volume to be created, either by external provisioner "block.csi.ibm.com" or manually created by system administrator
```

After some time ODF UI stopped showing capacity for FlashSystem

![image](https://user-images.githubusercontent.com/91637277/136197641-6306fa81-ffd7-4182-ae8a-cdbd80674216.png)

**Comment by shdn-ibm** [comment]
hi, @TymoT Please provide the cluster name and configuration for investigation?

**Comment by shdn-ibm** [comment]
waiting for reproduce. please hold the env after reproduced.

**Comment by TymoT** [comment]
@shdn-ibm issue recreated, details in https://jira.xiv.ibm.com/browse/ODF-65

**Comment by TymoT** [comment]
I managed to recreate this issue two times. The first time I thought the stats got back to normal after deleting too large pv (but the issue could as well dissapear on it's own). The second time the stats got back to normal after several minutes without any intervention. Than the issue didn't recreate in three trials at all. I tried once again today and couldn't recreate too. It looks this problem self heals on his own or doesn't recreate. I wold say this ticket could be closed @shdn-ibm unless you want to try something more.
